src/firefly/js/tables/TableRequestUtil.js

```diff
@@ -135,36 +135,6 @@ export function makeIrsaCatalogRequest(title, project, catalog, params={}, optio
     return omitBy(Object.assign(req, options, params, {id, tbl_id, META_INFO, UserTargetWorldPt, catalogProject, catalog}), isNil);
 }
 
-/**
- * creates the request to query LSST catalogs.  // TODO: more detail to be updated based on the LSST catalog DD content
- * @param {string} title    title to be displayed with this table result
- * @param {string} project
- * @param {string} catalog  the catalog name to search
- * @param {ConeParams|BoxParams|ElipParams} params   one of 'Cone','Eliptical','Box','Polygon','Table','AllSky'.
- * @param {TableRequest} [options]
- * @returns {TableRequest}
- * @func makeLsstCatalogRequest
- * @memberof firefly.util.table
- */
-export function makeLsstCatalogRequest(title, project, catalog, params={}, options={}) {
-    const req = {startIdx: 0, pageSize: 100};
-
-    title = title ?? catalog;
-    const tbl_id = options.tbl_id || uniqueTblId();
-    const id = get(params, 'SearchMethod')==='Table'?'LSSTMultiObjectSearch':'LSSTCatalogSearch';
-    const UserTargetWorldPt = params.UserTargetWorldPt || params.position;  // may need to convert to worldpt.
-    const table_name = catalog;
-    const meta_table = catalog;
-    const META_INFO = pickBy(Object.assign(options.META_INFO || {}, {title, tbl_id}));
-
-
-    options = omit(options, 'tbl_id');
-    params = omit(params, 'position');
-
-    return omitBy(Object.assign(req, options, params,
-                                {id, tbl_id, META_INFO, UserTargetWorldPt, table_name, meta_table, project}), isNil);
-}
-
 /**
  * creates the request to query VO catalogmakeLsstCatalogRequest
```

Comment:
`catalogmakeLsstCatalogRequest` this is probably a cut/paste error.  If you happen to do another commit, will you clean this up?